examples/solverdummy/clean.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -e -u
+
```

Comment:
```suggestion

rm -fv solverdummy
```

Reply:
Good observation!
That is covered by `make clean`.
We do not have any clear policy yet on whether clean scripts should be cleaning build artifacts as well. My current view for the tutorials is to not clean, but ignore. The motivation is that one typically only needs to build them once, and building sometimes takes time or is complicated / needs build dependencies that are not available.